tests/common.go

```diff
@@ -6624,3 +6625,123 @@ func AddMetadataDisk(n node.Node) error {
 	return nil
 
 }
+
+// generateRandomString Creates a random string of length n and returns it
+func generateRandomString(n int) string {
+	var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+	randomString := make([]rune, n)
+	for i := range randomString {
+		randomString[i] = letters[rand.Intn(len(letters))]
+	}
+	return string(randomString)
+}
+
+// createNamespaces Create N number of namespaces and return namespace list
+func createNamespaces(numberOfNamespaces int) ([]string, error) {
+
+	// Create multiple namespaces in string
+	var (
+		namespaces []string
+	)
+
+	// Create a good number of namespaces
+	for i := 0; i < numberOfNamespaces; i++ {
+		namespace := fmt.Sprintf("large-resource-%d-%v", i, time.Now().Unix())
+		nsName := &corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: namespace,
+			},
+		}
+		log.InfoD("Creating namespace %v", namespace)
+		_, err := k8sCore.CreateNamespace(nsName)
+		if err != nil {
+			return nil, err
+		} else {
+			namespaces = append(namespaces, namespace)
+		}
+	}
+	return namespaces, nil
+}
+
+// createConfigMaps with the given number of entries on specified namespaces
+func createConfigMaps(namespaces []string, numberOfConfigmap int, numberOfEntries int) error {
+
+	// Create random data to add in ConfigMap
+	var randomData = make(map[string]string)
+	for i := 0; i < numberOfEntries; i++ {
+		randomString := generateRandomString(80)
+		randomStringWithTimeStamp := fmt.Sprintf("%v.%v", randomString, time.Now().Unix())
+		randomData[randomStringWithTimeStamp] = randomString
+	}
+
+	// create the number of configmaps needed
+	for _, namespace := range namespaces {
+		for i := 0; i < numberOfConfigmap; i++ {
+			name := fmt.Sprintf("configmap-%s-%d-%v", namespace, i, time.Now().Unix())
+			log.InfoD("Creating Configmap: %v", name)
+			metaObj := metaV1.ObjectMeta{
+				Name:      name,
+				Namespace: namespace,
+			}
+			obj := &corev1.ConfigMap{
+				ObjectMeta: metaObj,
+				Data:       randomData,
+			}
+
+			k8sCore = core.Instance()
```

Comment:
k8score  variable can be initialized outside the loop. which can reduce unnecessary initilization for every iteration.

Reply:
Done